### ISSUE TITLE
`-v` option of `optuna study set-user-attr` command is not working

### ISSUE THREAD

**Issue by not522** [optuna/optuna#3483]
### Expected behavior

`-v` option as the abbreviation of `--value` is not working. `-v` option conflicts `--verbose` and `--value` in `optuna study set-user-attr`. We should remove `-v` for the abbreviation of `--value` and use it as the abbreviation of `--verbose` because it is `cliff`'s default option.

### Environment

- Optuna version: 3.0.0b1.dev
- Python version: 3.8.2
- OS: macOS-10.16-x86_64-i386-64bit
- (Optional) Other libraries and their versions:

### Error messages, stack traces, or logs

```shell
usage: optuna study set-user-attr [-h] [--study STUDY] [--study-name STUDY_NAME] --key KEY --value VALUE
optuna study set-user-attr: error: the following arguments are required: --value/-v
```

### Steps to reproduce

```sh
optuna create-study --storage sqlite:///test.db --study-name test-study
optuna study set-user-attr --storage sqlite:///test.db --study-name test-study --key test-key -v test
```

**Comment by himkt** [comment]
@nyanhi is working on the issue!

**Comment by hvy** [comment]
Related https://github.com/optuna/optuna/issues/3485

**Comment by hvy** [comment]
Let me close this issue as resolved by #3499. Feel free to reopen as needed.
